tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -1293,15 +1198,10 @@ async def test_set_nft_status(two_wallet_nodes: Any, trusted: Any) -> None:
     await server_0.start_client(PeerInfo("localhost", uint16(full_node_server._port)), None)
     await server_1.start_client(PeerInfo("localhost", uint16(full_node_server._port)), None)
 
-    for _ in range(1, num_blocks):
-        await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
-
-    funds = sum(
-        [calculate_pool_reward(uint32(i)) + calculate_base_farmer_reward(uint32(i)) for i in range(1, num_blocks - 1)]
-    )
+    await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
+    await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
 
-    await time_out_assert(20, wallet_0.get_unconfirmed_balance, funds)
```

Comment:
we don't care about asserting the unconfirmed balance anymore?

Reply:
we're just trying to figure out wallet is ready for minting and there are no transactions happening as we're minting first few blocks before (previous code was mainly copy/paste from tests for standard wallet afair)